script.js

```diff
@@ -1,50 +1,52 @@
 /* global MutationObserver */
 
-function updateStrategy () {
+function updateStrategy() {
   const branch = document.querySelector('.head-ref').textContent;
-  if (branch.startsWith('release/')) {
-    // do a MERGE if the branch is a release branch
-    // console.log('ONLY merge')
-    document.querySelector('.merge-message details button[value=merge]').click()
-    // hide the other buttons
+
+  if (branch === 'release') {
+    // For PRs targeting the release branch, use Squash and Merge
+    document.querySelector('.merge-message details button[value=squash]').click();
+    // Hide the other buttons
+    document.querySelector('.merge-message details button[value=merge]').style.visibility = "hidden";
+    document.querySelector('.merge-message details button[value=rebase]').style.visibility = "hidden";
+
+  } else if (branch.startsWith('rc/')) {
+    // For PRs targeting main from rc/* branches, use Merge Commit
+    document.querySelector('.merge-message details button[value=merge]').click();
+    // Hide the other buttons
     document.querySelector('.merge-message details button[value=squash]').style.visibility = "hidden";
     document.querySelector('.merge-message details button[value=rebase]').style.visibility = "hidden";
-  } else {
```

Comment:
Do we want to keep the else block for branches that don't match the pattern, leaving options open?

Reply:
We should leave options open imo, I would rather not block people